subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/provider/KotlinScriptEvaluator.kt

```diff
@@ -237,13 +237,14 @@ class StandardKotlinScriptEvaluator(
             scriptHost: KotlinScriptHost<*>,
             templateId: String,
             sourceHash: HashCode,
+            compilationClassPathHash: HashCode,
             parentClassLoader: ClassLoader,
```

Comment:
It is no longer used. Removed it.

Reply:
Actually, I'll probably revert this one back and add a feature flag for now where we could switch this feature off without reverting the whole thing in case we find trouble with it before 6.8